### PR TITLE
VEN-344 | Add free text field to berth model

### PR DESCRIPTION
**Pull request by a-rmz**
Add a free text field to the Berth model for comments.

### CODE REVIEW

File: resources/models.py [hunk 1]
```diff
@@ -406,6 +406,7 @@ class Berth(AbstractBoatPlace):
         related_name="berths",
         on_delete=models.PROTECT,
     )
+    comments = models.TextField(verbose_name=_("comments"), blank=True,)
```
Comment (igordavydsson) [inline resources/models.py]: the name should be `comment`, as field names in plural are used for reverse RoeignKeys and M2M relations 🙂

**Comment by igordavydsson** [comment]
please, avoid back-n-forth changes between commits. you could change the original migration file as well as `models.py` and then just amend your original commit. 
amending commits is the preferred way of reacting to PR feedback, unless you have a huge and complicated PR, where you have merged `develop` into you feature branch a few times.

**Comment by a-rmz** [comment]
> please, avoid back-n-forth changes between commits. you could change the original migration file as well as `models.py` and then just amend your original commit and `push -f`.
> amending commits is the preferred way of reacting to PR feedback, unless you have a huge and complicated PR, where you have merged `develop` into you feature branch a few times.

Okay! Good to know, wasn't sure if it was better to amend and `push -f` or add more commits, but will do from now.

**Comment by a-rmz** [comment]
Wohoo! 😄

**Pull request merged**